src/var_equality.jl

```diff
@@ -57,16 +57,19 @@ function Base.show(io::IOContext, t::VarianceEqualityTest)
     end
 end
 
-function anova(scores::AbstractVector{<:Real}...)
-    Nᵢ = [length(g) for g in scores]
+function anova(scores::AbstractVector{<:AbstractVector})
+    Nᵢ = length.(scores)
```

Comment:
No need for broadcasting here:
```suggestion
    Nᵢ = map(length, scores)
```

Reply:
Actually, `Nᵢ` has to be a vector -- that's what `VarianceEqualityTest` struct expects